xmake/core/main.lua

```diff
@@ -74,6 +85,11 @@ function main._help()
             utils.cprint(menu.copyright)
         end
 
+        -- print features
+        if menu.features then
```

Comment:
Seems right. but it helps us to test libraries' detection and know if a feature is enabled. I'm a little worry about detection :smiley:. Anyway I'll find a better way tomorrow. Maybe `xmake --version --verbose`?